omnibus/Gemfile.lock

```diff
@@ -144,18 +145,18 @@ GEM
     net-ssh (4.2.0)
     net-ssh-gateway (1.3.0)
       net-ssh (>= 2.6.5)
-    nio4r (2.2.0)
+    nio4r (2.3.0)
     nori (2.6.0)
     octokit (4.8.0)
       sawyer (~> 0.8.0, >= 0.5.3)
-    ohai (13.8.0)
-      chef-config (>= 12.5.0.alpha.1, < 14)
+    ohai (14.0.29)
```

Comment:
this?

Reply:
yeah I think we may need to pin in the Gemfile